include/openssl/err.h

```diff
@@ -284,7 +296,6 @@ unsigned long ERR_get_error(void);
  * additional data.
  */
 unsigned long ERR_get_error_line(const char **file, int *line);
-unsigned long ERR_get_error_func(const char **func);
```

Comment:
You must *replace* it, not just remove it, since it's a public function.  A macro would work, but lacks the deprecation attribute, so it would be preferable if the functions were properly deprecated, and you left an implementation that just returns zero.

Reply:
Oh hold on, they're *new* in master...  Never mind then!